packages/ripple-binary-codec/src/enums/definitions.json

```diff
@@ -981,6 +1032,26 @@
         "type": "Hash160"
       }
     ],
+    [
+      "TokenCurrency",
```

Comment:
Where can I find the reference for these definitions? (I don't see `TokenCurrency` mentioned in the spec proposal)

Reply:
They are internally used in rippled - https://github.com/gregtatcam/rippled/blob/266-deposit/src/ripple/app/tx/impl/AMMCreate.cpp#L213-L214

Reply:
IMO it's not ever worth reviewing definitions.json changes - it's derived directly from the rippled source code with xrpl-codec-gen, so you can trust it, and just make sure the tests look good.